src/detect-tcp-window.c

```diff
@@ -327,11 +306,10 @@ static int DetectWindowTestPacket01 (void)
                               {0, 1},
                               /* packet 2 should not match */
                               {0, 0} };
-    result = UTHGenericTest(p, 3, sigs, sid, (uint32_t *) results, 2);
+    FAIL_IF_NOT(UTHGenericTest(p, 3, sigs, sid, (uint32_t *) results, 2));
```

Comment:
It is better if we are explicit here with regard to which result we expect. If result is 0, we fail the test, if it's 1, we pass, right?

Reply:
i believe yes. I'll make the edit.